packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -912,7 +980,7 @@ export class Project extends ProjectBase {
       const keyStack = Stack.of(options.artifactBucket.encryptionKey);
       const projectStack = Stack.of(this);
       if (!(options.artifactBucket.encryptionKey instanceof kms.Key &&
-          (keyStack.account !== projectStack.account || keyStack.region !== projectStack.region))) {
+        (keyStack.account !== projectStack.account || keyStack.region !== projectStack.region))) {
```

Comment:
Let's leave this double-indent as-is.

Reply:
my editor 'fixed' these at some point. will revert